docs/modules/ROOT/examples/inference-service-explainer-lime.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "explainer-test-lime"
+  annotations:
+    sidecar.istio.io/inject: "true"
+    sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    serving.knative.openshift.io/enablePassthrough: "true"
```

Comment:
I think these annotations are necessary only for OpenShift Serverless + OpenShift Service Mesh not for upstream Istio (as far as I remember the default upstream is inject true) and Knative.

Btw this is a minor comment but you can mention in the document that has been tested using OpenShift + OpenShift Serverless + OpenShift Service Mesh and ODH

Reply:
Removed annotations from the KServe/Kubernetes example